### ISSUE TITLE
[Feature Request] Add a keyboard command for quick route selection on the map.

### ISSUE THREAD

**Issue by JohnMcChungus** [endless-sky/endless-sky#10540]
### Problem Description

While the game has extensive keyboard navigation, it seems to be missing the action of selecting a route on the map with a single key press after using the Find feature. "Find" already targets your destination, but there's no keyboard button to select it as your new destination.

This feature request is also about bringing additional control parity to the game, as you can create custom and shortest-distance routes with the mouse (clicking vs. shift clicking), but you can _only_ create custom routes with the keyboard.

### Related Issue Links

I couldn't find any.

### Desired Solution

Press Enter (or another key) after searching for a star with the Find feature to select it as a new jump destination.

### Alternative Approaches

Alternatively, route selection can simply occur immediately after highlighting a destination using Find, though the automatic switching of your route with each search may be annoying. A Map setting (by which I mean Preferences > Settings > Map) could be used to toggle between automatic route switching and having to press Enter to manually select the target as a new route.

### Additional Context

As it stands, the only way to currently set a route on the map with a keyboard is to tediously use Shift+J to add a new "Segment", press J to cycle the jump destination until you get the correct one, then repeat this. To remove a segment, you must press Backspace. This process is unacceptable for any distance longer than a few jumps, and may not result in the most efficient route, wasting time and fuel with erroneous jumps.

**Comment by TomGoodIdea** [comment]
duplicate of #5981
